cmd/start.go

```diff
@@ -52,6 +53,7 @@ func init() {
 	flags.StringP(
 		bearerTokenFlagName, "b", "", "Set a bearer token to use for remote sync")
 	flags.StringSliceP(corsFlagName, "C", []string{}, "CORS allowed origins, * will allow all origins")
+	flags.Bool(NoopRequestLoggingFlagName, false, "Disable all logging levels pertaining to flag evaluation requests")
```

Comment:
Can we use something a bit more common such as `debug` and when that's enabled logs are on?
We then use a `v` verbosity flag like programs e.g. kubectl?

```
flagd start -f ... --debug --v=5
```